atom/browser/ui/inspectable_web_contents_impl.cc

```diff
@@ -59,9 +59,9 @@ const char kChromeUIDevToolsRemoteFrontendBase[] =
     "https://chrome-devtools-frontend.appspot.com/";
 const char kChromeUIDevToolsRemoteFrontendPath[] = "serve_file";
 
-const char kDevToolsBoundsPref[] = "brightray.devtools.bounds";
-const char kDevToolsZoomPref[] = "brightray.devtools.zoom";
-const char kDevToolsPreferences[] = "brightray.devtools.preferences";
+const char kDevToolsBoundsPref[] = "electron.devtools.bounds";
```

Comment:
The key name changes will break existing preferences, but since it will be in next major release, guess its fine.

Reply:
That and these are purely used for devtools

Reply:
The first change is for generation of media device id, but since its generated from a random value. We can freely change the name.

And the rest are devtools related. But even if its devtools related api, I don't think we can break them unless its in a major/minor release ? They are not a blocker for this PR, just wanted to highlight the result of some changes.

Reply:
Yeah was just agreeing we can break these in the next major easily because they're barely app facing just being keys for devtools prefs